_pkgdown.yml

```diff
@@ -0,0 +1,12 @@
+url: evaluate.r-lib.org
+
+template:
+  package: tidytemplate
+  bootstrap: 5
+  
+  includes:
+    in_header: |
+      <script defer data-domain="evaluate.r-lib.org,all.tidyverse.org" src="https://plausible.io/js/plausible.js"></script>  
+      
+development:
+  mode: auto
```

Comment:
Will you switch to usual tidyverse / r-lib numbering ? 

Not sure about the evolution of auto mode, but in R Markdown ecosystem with the other way to number with each release being x.y, and dev being x.y.z, this was not working. 

So we have only a website built on main for rmarkdown and others. 

I am ok with switching the version numbering, just so it is clear.

Reply:
Yes, plan is to switch to usual numbering.

Reply:
Also it feels like it's probably time for 1.0.0 release.